src/pymovements/transforms.py

```diff
@@ -91,8 +97,10 @@ def pix2deg(
     distance_px = distance_cm * (screen_px / screen_cm)
 
     # center screen coordinates such that 0 is in the center of the screen
```

Comment:
I think we should delete this comment or adjust it.

Reply:
yes, that leftover doesn't make any sense. I'll adjust it.